charts/rancher-aks-operator-crd/rancher-aks-operator-crd/1.0.0+up1.0.1-rc11/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
```

Comment:
Believe you need to delete the old rc's charts and assets, and update index.yaml

Reply:
Ok, thank you I have updated it according to your instruction